### PR TITLE
Enhance/Fix : Firmware/LLDPNeighbor/Users

### PR DESCRIPTION
**Pull request by alagoutte**
Add missing Tests
Enhance Example and Documentation

### CODE REVIEW

File: PowerArubaCX/Public/Firmware.ps1 [hunk 1]
```diff
@@ -8,12 +14,19 @@
         Get all informations about Aruba CX firmware
 
         .EXAMPLE
-        Get-ArubaCXfirmware
+        Get-ArubaCXFirmware
 
         Get all informations about Aruba CX firmware, first image an secondary image
```
Comment (CedricMoreau) [inline PowerArubaCX/Public/Firmware.ps1]: an --> and

File: PowerArubaCX/Public/Users.ps1 [hunk 2]
```diff
@@ -8,13 +14,23 @@
...
         .EXAMPLE
-        Get-ArubaCXUser
+        Get-ArubaCXUsers
 
         List all users in ArubaCX
 
+        .EXAMPLE
+        Get-ArubaCXUsers -user admin
+
+        list info about user admin
```
Comment (CedricMoreau) [inline PowerArubaCX/Public/Users.ps1]: Caps ?

File: README.md [hunk 3]
```diff
@@ -442,6 +442,128 @@ You can create a new interface (LAG, Loopback, Vlans) `Add-ArubaCXInterfaces`, r
...
+
+#Get LLDP Neighbor of port 1/1/8 with depth 2
+
+    Get-ArubaCXLLDPNeighbor 1/1/8 -depth 2
+
+    08:00:09:0d:2d:06,1/1/9
+    -----------------------
+    @{chassis_id=08:00:09:0d:2d:06; dcbx_cee_status=; mac_addr=08:00:09:0d:2d:06; med_network_policy=; neighbor_info=; neighbor_info_organizational=; port_id=1/1/9}
+
+#for Get all info, you can use
```
Comment (CedricMoreau) [inline README.md]: for --> to

File: README.md [hunk 4]
```diff
@@ -442,6 +442,128 @@ You can create a new interface (LAG, Loopback, Vlans) `Add-ArubaCXInterfaces`, r
...
+    power_requested_pair_a       : 0
+    power_requested_pair_b       : 0
+    power_source                 : Unknown
+    power_supported              : 1
+    remote_index                 : 2
+
+```
+
+### Firmware
+For Get Firmware information on Aruba CX Switch
```
Comment (CedricMoreau) [inline README.md]: For --> to

File: Tests/integration/Users.Tests.ps1 [hunk 5]
```diff
@@ -0,0 +1,113 @@
+#
+# Copyright 2022, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+. ../common.ps1
+
```
Comment (CedricMoreau) [inline Tests/integration/Users.Tests.ps1:7]: Necessary blank line ?

File: Tests/integration/Users.Tests.ps1 [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+Describe "Get Users" {
+    It "Get Users Does Not Throw an error" {
+        {
+            Get-ArubaCXUsers
+        } | Should -Not -Throw
+    }
+
+    #It "Get Users (admin) and confirm (via Confirm-ArubaCXUsers)" {
```
Comment (CedricMoreau) [inline Tests/integration/Users.Tests.ps1:19]: Why leave it commented ? Maybe use <# #> ?

Comment (alagoutte) [reply Tests/integration/Users.Tests.ps1:19]: for futur use when add support of Add/Set/Remove Users...

**Pull request merged**